### PR TITLE
Updating JA translation of ASA content

### PR DESCRIPTION
**Pull request by paulwallace-okta**
Here's the deploy preview link to the ASA JA section: https://deploy-preview-478--extraordinary-horse-71260e.netlify.app/asa/ja-jp/content/topics/adv_server_access/docs/asa-overview

### CODE REVIEW

File: asa/ja-jp/Content/Topics/Adv_Server_Access/docs/about-asa.htm [hunk 1]
```diff
@@ -1,272 +1,272 @@
...
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="replace_top_nav">
+                                        <div class="navbar">
+                                            <div class="dropdown" data-mc-conditions="MultiProdPubish.DocMenu">                                                 <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>                                                 <div class="dropdown-content">
+                                                    <div><a href="https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm">リリースノート</a>                                                     </div>
+                                                    <div><a href="https://help.okta.com/ja-jp/Content/index.htm">製品ドキュメント</a>                                                     </div>
```
Comment (mauriciocastillosilva-okta) [inline asa/ja-jp/Content/Topics/Adv_Server_Access/docs/about-asa.htm:145]: this changes we should automate, we have https://oktainc.atlassian.net/browse/OKTA-540003 for it

File: asa/ja-jp/Content/Topics/Adv_Server_Access/docs/ad-user-manage.htm [hunk 2]
```diff
@@ -250,11 +250,11 @@ <h3>Active DirectoryパスワードなしID</h3>
...
                                                         </div>
                                                         <div>
                                                             <h2><a name="assign-users-or-groups-to-ASA"></a>ユーザーまたはグループを<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span><br />に割り当てる</h2>
-                                                            <p>前の手順を終えると、属性の構成が完了し、使用を開始できます。<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>に割り当てられたユーザーには、これらの属性が割り当てられます。「<a href="https://help.okta.com/okta_help.htm?type=oie&amp;id=csh-assign-apps" target="_blank">アプリケーションのユーザーへの割り当て</a>」をご覧ください。</p>
+                                                            <p>前の手順を終えると、属性の構成が完了し、使用を開始できます。<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>に割り当てられたユーザーには、これらの属性が割り当てられます。「<a href="../../../../../../okta_help.htm?locale=ja-JP&amp;type=oie&amp;id=csh-assign-apps" target="_blank">アプリケーションのユーザーへの割り当て</a>」をご覧ください。</p>
```
Comment (mauriciocastillosilva-okta) [inline asa/ja-jp/Content/Topics/Adv_Server_Access/docs/ad-user-manage.htm]: This changes should be automated, https://oktainc.atlassian.net/browse/OKTA-538298 should take care of this

File: asa/ja-jp/Content/Topics/ReleaseNotes/advanced-server-access-release-notes.htm [hunk 3]
```diff
@@ -328,6 +328,8 @@ <h1 class="relnotes">未翻訳のドキュメント</h1>
                                     </div>
                                     <!-- ********************** Scripts ************************************* -->
                                     <!-- Foundation -->
+                                    <script src="../../Resources/Scripts/js/ignore-dompurify.js">
```
Comment (mauriciocastillosilva-okta) [inline asa/ja-jp/Content/Topics/ReleaseNotes/advanced-server-access-release-notes.htm]: https://oktainc.atlassian.net/browse/OKTA-538286 should take care of automating this (if needed)

File: asa/ja-jp/Content/Topics/Adv_Server_Access/docs/asa-overview.htm [hunk 4]
```diff
@@ -195,25 +195,25 @@ <h1>アドバンストサーバーアクセス</h1>
                                                             <p class="tile-subtitle"><span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>のチームをセットアップし、<span class="mc-variable okta-variables.ProductName variable">Okta</span>からユーザーとグループをプッシュします。</p>
                                                         </div>
                                                         <div>
-                                                            <p class="tile-title">サーバーエージェント</p>
+                                                            <p class="tile-title"><a href="sftd.htm">サーバーエージェント</a></p>
```
Comment (mauriciocastillosilva-okta) [inline asa/ja-jp/Content/Topics/Adv_Server_Access/docs/asa-overview.htm]: What happened here? Did XTM removed the `<a>` elements or the version we sent did not have it?

**Review by mauriciocastillosilva-okta — APPROVED** [review]
looks good, I added some comments to point to jiras to automate all these issues. And I left one question to be sure what happened

File: asa/ja-jp/Content/Topics/ReleaseNotes/advanced-server-access-release-notes.htm (hunk 3 above)
Comment (paulwallace-okta) [reply asa/ja-jp/Content/Topics/ReleaseNotes/advanced-server-access-release-notes.htm]: In this case, I believe we added the `ignore-dompurify.js` script to the EN doc after we sent this to translation. So, it'll be automatically included in future rounds.

File: asa/ja-jp/Content/Topics/Adv_Server_Access/docs/asa-overview.htm (hunk 4 above)
Comment (paulwallace-okta) [reply asa/ja-jp/Content/Topics/Adv_Server_Access/docs/asa-overview.htm]: We had missing links in the tiles at the time that we sent the project out (the issue was in source EN, it's been fixed).

**Pull request merged**